### PR TITLE
Setup translation infrastructure

### PR DESCRIPTION
**Pull request by ofedoren**

### CODE REVIEW

File: lib/hammer_cli_foreman_webhooks/i18n.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+require 'hammer_cli/i18n'
+
+module HammerCLIForemanWebhooks
+  module I18n
+    class LocaleDomain < HammerCLI::I18n::LocaleDomain
+      def translated_files
+        Dir.glob(File.join(File.dirname(__FILE__), '../**/*.rb'))
```
Comment (ekohl) [inline lib/hammer_cli_foreman_webhooks/i18n.rb]:
```suggestion
        Dir.glob(File.join(__dir__, '..', **', '*.rb'))
```

File: lib/hammer_cli_foreman_webhooks/i18n.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      end
+
+      def locale_dir
+        File.join(File.dirname(__FILE__), '../../locale')
```
Comment (ekohl) [inline lib/hammer_cli_foreman_webhooks/i18n.rb]:
```suggestion
        File.join(__dir__, '..', '..', 'locale')
```

**Review by ekohl — COMMENTED** [review]
Looks good. Minor notes inline: `File.dirname(__FILE__)` is equal to `__dir__`.

**Comment by ofedoren** [comment]
Thanks, @ekohl!

**Pull request merged**